db/migrate/20230228164831_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :bio
+      t.string :photo
+
+      t.timestamps
```

Comment:
- [ ] You did a good job however, according to the Erd diagram you have to add PostsCounter to the user table.